docs/how_to_guides/creating_and_editing_expectations/how_to_create_custom_expectations_for_sqlalchemy.rst

```diff
@@ -3,16 +3,266 @@
+            CustomSqlAlchemyDataset,
+        )
+
+5. **Test your Expectations**
+
+    At this point, you can test your new Expectations exactly like built-in Expectations. All out-of-the-box Expectations will still be available, plus your new methods.
+
+    .. code-block:: python
+
+        my_data_asset.expect_column_values_to_be_even("ReportsTo")
+
+    returns
+
+    .. code-block:: json
+
+        {
+            "exception_info": null,
+            "success": false,
+            "result": {
+                "element_count": 8,
+                "missing_count": 1,
+                "missing_percent": 12.5,
+                "unexpected_count": 2,
+                "unexpected_percent": 25.0,
+                "unexpected_percent_nonmissing": 28.57142857142857,
+                "partial_unexpected_list": [
+                    1,
+                    1
+                ]
+            },
+            "meta": {}
+        }
+
+    As mentioned previously, that the ``column_map_expectation`` decorator extends the arguments to include other arguments, like ``mostly``. Please see the module documentation for full details.
+
+    .. code-block:: python
+
+        my_df.expect_column_values_to_be_even("ReportsTo", mostly=.7)
+
+    returns
+
+    .. code-block:: json
+
+        {
+            "exception_info": null,
+            "success": true,
+            "result": {
+                "element_count": 8,
+                "missing_count": 1,
+                "missing_percent": 12.5,
+                "unexpected_count": 2,
+                "unexpected_percent": 25.0,
+                "unexpected_percent_nonmissing": 28.57142857142857,
+                "partial_unexpected_list": [
+                    1,
+                    1
+                ]
+            },
+            "meta": {}
+        }
+
+    .. code-block:: python
+
+        my_batch.expect_column_mode_to_equal("ReportsTo", value=2)
+
+    returns
+
+    .. code-block:: json
+
+        {
+            "exception_info": null,
+            "success": true,
+            "result": {
+                "observed_value": 2,
+                "element_count": 8,
+                "missing_count": 1,
+                "missing_percent": 12.5
+            },
+            "meta": {}
+        }
+
+    Often, the best development loop for custom Expectations is iterative: editing Expectations in ``MyCustomSqlAlchemyDataset``, then re-running the cells to load data and execute Expectations on data.
+
+Additional notes
+----------------
+
+
+Other Expectation decorators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aside from ``column_map_expectations``, there are several other types of Expectations you can create.
+
+- ``column_aggregate_expectations`` generate a single observed value for a whole column.
+- ``column_pair_map_`` and ``column_pair_aggregate_expectations`` apply to pairs of columns.
+- ``multicolumn_map_`` and ``multicolumn_aggregate_expectations`` apply to multiple columns.
```

Comment:
Not implemented

```suggestion
```

Reply:
Good point. We should fix that. :)

This block was copied directly from the custom pandas Expectation article. It's exhibit A for content that belongs in the Reference section, not here.